cloudant-sync-datastore-core/src/main/java/com/cloudant/sync/replication/ReplicationPolicyManager.java

```diff
@@ -20,16 +20,31 @@
 import java.util.Arrays;
 import java.util.List;
 
+/**
+ * Class to manage replication lifecycle events.
```

Comment:
Maybe `Manages the coordination of starting/stopping all replicators used in a replication policy and providing monitoring of when replications have completed.`